test/lib/users/users_database_test.rb

```diff
@@ -21,66 +21,102 @@
 require "users/users_database"
 
 describe Users::UsersDatabase do
+  let(:databases) { described_class.all }
+  let(:fixture_root) { FIXTURES_PATH.join("root") }
+  let(:fixture_root2) { FIXTURES_PATH.join("root2") }
+  let(:fixture_root3) { FIXTURES_PATH.join("root3") }
+
+  before do
+    Users::UsersDatabase.all.clear
+
+    # Mock access time: files in root2 are more recent than files in root and root3
+    allow(File).to receive(:atime) do |path|
+      path =~ /root2/ ? Time.now : Time.now - 1200
+    end
+  end
+
+  describe ".all" do
+    let(:databases) { described_class.all }
+
+    before do
+      Users::UsersDatabase.import(fixture_root)
+      Users::UsersDatabase.import(fixture_root2)
+      Users::UsersDatabase.import(fixture_root3)
+    end
+
+    it "returns found users databases" do
+      expect(databases.size).to eq(3)
+    end
+
+    it "returns the most recent accesed first" do
+      expect(databases.first.passwd).to start_with("a_user")
+    end
+  end
+
   describe ".import" do
+    let(:databases) { described_class.all }
+
     it "always stores databases sorted by access time" do
-      # Mock access time: files in root2 are more recent than files in root
-      allow(File).to receive(:atime) do |path|
-        path =~ /root2/ ? Time.now : Time.now - 1200
-      end
+      databases.clear
+      expect(databases).to be_empty
+
+      Users::UsersDatabase.import(fixture_root)
+      Users::UsersDatabase.import(fixture_root2)
+      expect(databases.size).to eq(2)
+      expect(databases.first.passwd).to start_with("a_user")
 
-      Users::UsersDatabase.all.clear
-      expect(Users::UsersDatabase.all).to be_empty
-      Users::UsersDatabase.import(FIXTURES_PATH.join("root"))
-      Users::UsersDatabase.import(FIXTURES_PATH.join("root2"))
-      databases = Users::UsersDatabase.all
-      expect(databases.size).to eq 2
-      expect(databases.first.passwd).to start_with "a_user"
-
-      Users::UsersDatabase.all.clear
-      expect(Users::UsersDatabase.all).to be_empty
-      Users::UsersDatabase.import(FIXTURES_PATH.join("root2"))
-      Users::UsersDatabase.import(FIXTURES_PATH.join("root"))
-      databases = Users::UsersDatabase.all
-      expect(databases.size).to eq 2
-      expect(databases.first.passwd).to start_with "a_user"
+      databases.clear
+      expect(databases).to be_empty
+
+      Users::UsersDatabase.import(fixture_root2)
+      Users::UsersDatabase.import(fixture_root3)
+      Users::UsersDatabase.import(fixture_root)
+      expect(databases.size).to eq(3)
+      expect(databases.first.passwd).to start_with("a_user")
```

Comment:
In both cases the most recent is the same. I would try to do an example where the result is different.